docs/src/solvers/pinn.md

```diff
@@ -55,13 +55,19 @@ List of training strategies that are available now:
    to the spacing per direction.
  - `StochasticTraining(points)`: `points` number of sochastically sampled points from the domain. 
    In each optimization iteration, we randomly select a new subset of points from a full training set.
-- `QuasiRandomTraining(sampling_alg)`: The training set is generated on quasi-random low discrepency
-  sequences. See the [QuasiMonteCarlo.jl](https://github.com/SciML/QuasiMonteCarlo.jl) for the full
+- `QuasiRandomTraining(sampling_alg,points,minibatch)`: The training set is generated on quasi-random low discrepency
+  sequences.
+  `sampling_alg`is the quasi-Monte Carlo sampling algorithm,
+  `points` is number of quasi-random points in training set,
+  `minibatch` is number of generated training samples.
+  See the [QuasiMonteCarlo.jl](https://github.com/SciML/QuasiMonteCarlo.jl) for the full
   set of quasi-random sampling algorithms which are available.
-- `QuadratureTraining(quadrature_alg)`: The loss is computed as an approximation of the integral of the PDE loss 
+- `QuadratureTraining(quadrature_alg,reltol,abstol,maxiters,batch)`: The loss is computed as an approximation of the integral of the PDE loss
```

Comment:
it's probably reasonable to do reltol and abstol as kwargs, because `abstol=1e-6,reltol=1e-3` should essentially be as accurate as Float32 accuracy can get. Maxiters could probably have an absurd default like `Int(1e5)`, mostly to check for divergence. It's usually adaptive algorithms that get defaults like this because those defaults are probably good for almost every case.